central/scannerdefinitions/handler/handler.go

```diff
@@ -461,6 +483,60 @@ func (h *httpHandler) openMostRecentV4File(t updaterType, updaterKey, fileName s
 	return file, nil
 }
 
+// openMostRecentV4OfflineFile gets desired offline file from compressed bundle: offlineScannerV4DefinitionBlobName
+func (h *httpHandler) openMostRecentV4OfflineFile(ctx context.Context, t updaterType, updaterKey, fileName string) (file *vulDefFile, err error) {
```

Comment:
nit: it doesn't seem like we need to name the return variables here. Let's remove that